rpm/netavark.spec

```diff
@@ -22,6 +22,12 @@
 # Minimum X.Y dep for aardvark-dns
 %define major_minor %((v=%{version}; echo ${v%.*}))
 
+%if %{defined rhel} && 0%{?rhel} >= 10
```

Comment:
just to confirm the rhel macro is also defined on centos stream?

Reply:
yes, %{rhel} is also on centos as well as Fedora ELN. But %{centos} is only on centos stream

Reply:
i can add a comment in spec if it helps

Reply:
done. PTAL